Scripts/Services/NaturalLanguageUnderstanding/v1/NaturalLanguageUnderstanding.cs

```diff
@@ -93,6 +94,65 @@ public bool DisableSslVerification
         #endregion
 
         #region Constructor
+        /// <summary>
+        /// NaturalLanguageUnderstanding constructor. Use this constructor to auto load credentials via ibm-credentials.env file.
+        /// </summary>
+        public NaturalLanguageUnderstanding()
+        {
+            var credentialsPaths = Utility.GetCredentialsPaths();
+            if (credentialsPaths.Count > 0)
+            {
+                string ApiKey = "";
+                string Endpoint = "";
+                string Username = "";
+                string Password = "";
+
+                foreach (string path in credentialsPaths)
+                {
+                    if (Utility.LoadEnvFile(path))
+                    {
+                        break;
+                    }
+                }
+
+                string apiKey = Environment.GetEnvironmentVariable(ServiceId.ToUpper() + "_APIKEY");
+                if (!string.IsNullOrEmpty(apiKey))
+                    ApiKey = apiKey;
+                string un = Environment.GetEnvironmentVariable(ServiceId.ToUpper() + "_USERNAME");
+                if (!string.IsNullOrEmpty(un))
```

Comment:
what do you think about using `username` and `password` instead of `un` and `pw`?

Reply:
hah I was using `username` and `password` at first but those are already properties of the service class. When the properties are set we check if it has bad characters `{}""` but get a null pointer if there is no env variable. I can definitely change this though to put a null check in the check for bad characters if you think its necessary

Reply:
that approach would be better I think.

Reply:
Done!